packages/terra-application-links/src/list/ApplicationList.scss

```diff
@@ -4,9 +4,7 @@
     color: var(--terra-application-list-color, #1c1f21);
     display: block;
     font-size: var(--terra-application-list-font-size, 1.071rem);
-    line-height: var(--terra-application-list-line-height, 3.214rem);
-    padding-left: var(--terra-application-list-padding-left 1.2143rem);
-    padding-right: var(--terra-application-list-padding-right, 0.7143rem);
+    padding: var(--terra-application-list-padding, 0.8225rem 0.7143rem 0.8225rem 1.2143rem);
```

Comment:
This will work fine in RTL since both left and right paddings are equal in this default theme. There may be a concern in other themes, if the left and right padding are not equal, not entirely sure how the PostCSS RTL plugin will handle this.

Reply:
Would you suggest just splitting the props into 4?

Reply:
These two aren't equal left and right values.

Reply:
If it becomes an issue, we can split padding into into individual props. I'm not too worried about it.

Reply:
Since it would be a non-passive theming change i was wondering if we should just do it now before release.

Reply:
I can make another pr for it, that won't be as priority to get in.